include/lsst/afw/image/Exposure.h

```diff
@@ -150,9 +150,9 @@ class Exposure : public lsst::daf::base::Persistable,
 
     // Get Members
     /// Return the MaskedImage
-    MaskedImageT getMaskedImage() { return _maskedImage; }
+    MaskedImageT & getMaskedImage() { return _maskedImage; }
     /// Return the MaskedImage
-    MaskedImageT getMaskedImage() const { return _maskedImage; }
+    MaskedImageT const & getMaskedImage() const { return _maskedImage; }
```

Comment:
The fact that it didn't return an internal handle before was a workaround for Swig wrapping this into a Python object that didn't hold a reference to the Exposure to keep it alive, yielding dangling pointers.